Emby.Server.Implementations/Playlists/ManualPlaylistsFolder.cs

```diff
@@ -49,5 +49,10 @@ protected override QueryResult<BaseItem> GetItemsInternal(InternalItemsQuery que
             query.Parent = null;
             return LibraryManager.GetItemsResult(query);
         }
+
+        public override string GetClientTypeName()
+        {
+            return "ManualPlaylistsFolder";
```

Comment:
Use `nameof`?

Reply:
The class is `PlaylistsFolder` for some reason, so I don't see how that would help

Reply:
Analyzers should be complaining that the fine name and class don't match up :thinking:

Reply:
It is, but that wasn't the scope of this PR. If you want I can change the filename as well.